.github/workflows/gradle.yml

```diff
@@ -855,158 +863,207 @@ jobs:
           label: ${{ needs.start-kube-acceptance-test-runner.outputs.label }}
           ec2-instance-id: ${{ needs.start-kube-acceptance-test-runner.outputs.ec2-instance-id }}
 
-  ## Kube Acceptance Tests
-  # Docker acceptance tests run as part of the build job.
-  # In case of self-hosted EC2 errors, remove this block.
-  # TODO:
-  # Kyryl to fix issue
-  # start-helm-acceptance-test-runner:
-  #   name: "Platform: Start Helm Acceptance Test Runner"
-  #   needs:
-  #     - changes
-  #     - find_valid_pat
-  #   # Because scheduled builds on master require us to skip the changes job. Use always() to force this to run on master.
-  #   if: needs.changes.outputs.backend == 'true' || needs.changes.outputs.build == 'true' || (always() && github.ref == 'refs/heads/master')
-  #   timeout-minutes: 10
-  #   runs-on: ubuntu-latest
-  #   outputs:
-  #     label: ${{ steps.start-ec2-runner.outputs.label }}
-  #     ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
-  #   steps:
-  #     - name: Checkout Airbyte
-  #       uses: actions/checkout@v3
-  #     - name: Start AWS Runner
-  #       id: start-ec2-runner
-  #       uses: ./.github/actions/start-aws-runner
-  #       with:
-  #         # github-self-hosted-runner-ubuntu-20-with-150gdisk-docker-20.10.7-and-socat
-  #         ec2-image-id: ami-0c1a9bc22624339d8
-  #         aws-access-key-id: ${{ secrets.SELF_RUNNER_AWS_ACCESS_KEY_ID }}
-  #         aws-secret-access-key: ${{ secrets.SELF_RUNNER_AWS_SECRET_ACCESS_KEY }}
-  #         github-token: ${{ needs.find_valid_pat.outputs.pat }}
+
+  # ## Kube Acceptance Tests
+  # # Docker acceptance tests run as part of the build job.
+  # # In case of self-hosted EC2 errors, remove this block.
+  # # TODO:
+  # # Kyryl to fix issue
+  start-helm-acceptance-test-runner:
+    name: "Platform: Start Helm Acceptance Test Runner"
+    needs:
+      - changes
+      - find_valid_pat
+    # Because scheduled builds on master require us to skip the changes job. Use always() to force this to run on master.
+    if: needs.changes.outputs.backend == 'true' || needs.changes.outputs.build == 'true' || (always() && github.ref == 'refs/heads/master')
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    outputs:
+      label: ${{ steps.start-ec2-runner.outputs.label }}
+      ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - name: Start AWS Runner
+        id: start-ec2-runner
+        uses: ./.github/actions/start-aws-runner
+        with:
+          # github-self-hosted-runner-ubuntu-20-with-150gdisk-docker-20.10.7-and-socat
+          ec2-image-id: ami-0c1a9bc22624339d8
```

Comment:
do we need a different ami from the default one set in the action?
https://github.com/airbytehq/airbyte/blob/34d4a71875e5413cee063fd301dc9b406686238b/.github/actions/start-aws-runner/action.yml#L12
If so, what's different about this ami?

Reply:
https://github.com/airbytehq/airbyte/blob/master/.github/workflows/gradle.yml#L685